File: VB/CustomizeTemplate.Module/Module.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating


Namespace CustomizeTemplate.Module
    Public NotInheritable Partial Class CustomizeTemplateModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()
        End Sub
        Public Overrides Function GetModuleUpdaters(ByVal objectSpace As IObjectSpace, ByVal versionFromDB As Version) As IEnumerable(Of ModuleUpdater)
            Dim updater As ModuleUpdater = New DatabaseUpdate.Updater(objectSpace, versionFromDB)
            Return New ModuleUpdater() { updater }
        End Function
    End Class
End Namespace

File: CS/CustomizeTemplate.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;


namespace CustomizeTemplate.Module {
    public sealed partial class CustomizeTemplateModule : ModuleBase {
        public CustomizeTemplateModule() {
            InitializeComponent();
        }
        public override IEnumerable<ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
            ModuleUpdater updater = new DatabaseUpdate.Updater(objectSpace, versionFromDB);
            return new ModuleUpdater[] { updater };
        }
    }
}
